### ISSUE TITLE
[Question] Automerge changes from `release/*` to `master`

### ISSUE THREAD

**Issue by jeffchew** [Logerfo/gitflow-action#17]
I'm new with setting up Github actions. I'd like to set up a flow where any changes to `release/*` gets merged down to `master` automatically. Would this configuration work with your action:

```yml
name: Gitflow
on: 
  push:
  pull_request_review:
  check_run:
    types: [completed]

jobs:
  build:
    name: Gitflow
    runs-on: ubuntu-16.04
    steps:
    - uses: Logerfo/gitflow-action@0.0.4
      with:
        github-token: ${{ secrets.GITHUB_TOKEN }}
        release: release/*
        master: master
        label: gitflow
        auto-merge: true
        require-merge: false
```

**Comment by Logerfo** [comment]
Currently, it's not possible to use branch name patterns, only constants. Here's what the branch configurations are used for:
- `dev`: only used as a target, pattern would not make sense;
- `master`: used as a target (from `release`) **and** as head (to `dev`), pattern would allow more complex configurations for head, but it would break the target;
- `release`: only used as head, pattern would allow more complex configurations without breaking anything.

Summing up: patterns will **never** be supported for the `dev` branch, they **can** be supported for the release branch and, if split into two configurations (head and target), `master` also **could** support them.
I don't see myself developing this in the near future, but anyone should feel free to send a pull request to achieve this and I'll be glad to release it. This is being tracked by #18.
